src/invoker/src/handler.rs

```diff
@@ -0,0 +1,117 @@
+use crate::{
+    api::{InvokeRequest, InvokeResponse, Output},
+    executor::{Executor, SandboxGlobalSettings},
+    graph_interp::Interpreter,
+    print_invoke_request::PrintWrapper,
+};
+use anyhow::Context as _;
+use minion::erased::Backend;
+use std::path::PathBuf;
+
+pub struct HandlerConfig {
+    pub work_dir: PathBuf,
+}
+
+pub struct Handler {
+    cfg: HandlerConfig,
+    sandbox_global_settings: SandboxGlobalSettings,
+    minion_backend: Box<dyn Backend>,
+}
+
+fn setup_minion() -> anyhow::Result<Box<dyn Backend>> {
+    let mut settings = minion::linux::Settings::new();
+    settings.cgroup_prefix = "/jjs".into();
```

Comment:
What if steps using different sandboxes are executed at the same time?

Reply:
Minion creates a new cgroup under this prefix for each sandbox.